examples/xlio_ultra_api_ping_pong.c

```diff
@@ -15,6 +15,22 @@
  * Usage (help)  : ./examples/xlio_ultra_api_ping_pong --help
  * Usage (server): LD_PRELOAD=libxlio.so ./examples/xlio_ultra_api_ping_pong -s -i 192.168.1.100
  * Usage (client): LD_PRELOAD=libxlio.so ./examples/xlio_ultra_api_ping_pong -c -i 192.168.1.100
+ *
+ * Code Structure
+ * ==============
+ * The main XLIO Ultra API flow is located at the top of the file. Auxiliary code follows below.
+ * Main parts:
+ *  - XLIO Ultra API initialization and resources management
```

Comment:
XLIO Ultra API initialization and resources management ->
XLIO Ultra API initialization and resource management